examples/aligning_aia_with_eis_maps.py

```diff
@@ -0,0 +1,67 @@
+"""
+===================================================
+Coaligning AIA and EIS data with cross-correlations
+===================================================
+
+This example shows how to coalign AIA and EIS data using the method of cross-correlations which is implemented in the match_template function within the sunkit-image.
+Here we have an EIS raster image which has an incorrect pointing. This would be corrected using the AIA image as a reference. 
+
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from sunpy.net import Fido ,attrs as a
+import sunpy.map
+from sunkit_image.coalignment import coalignment
+from sunkit_image.data.test import get_test_filepath
+
+###################################################################################
+# Firstly, let's load up the EIS raster and see what it looks like.
+# This raster in particular doesn't have the correct pointing information.
+
+eis_map = sunpy.map.Map(get_test_filepath("eis_20140108_095727.fe_12_195_119.2c-0.int.fits"))
+eis_map.plot()
```

Comment:
I hope we can replace this with a FIDO query.

Reply:
Unfortunately that's not going to happen.

Reply:
That is a shame.